### PR TITLE
docs: fix linter errors for what-is-angular.md

### PR DESCRIPTION
**Pull request by dsnoeijer**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #47179

## What is the new behavior?
No more linter errors in the file.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/what-is-angular.md [hunk 1]
```diff
@@ -53,18 +53,22 @@ When Angular renders this component, the resulting DOM looks like this:
...
 
+<!-- vale Angular.Google_WordListWarnings = NO -->
+
 Every component has an HTML template that declares how that component renders.
 You define this template either inline or by file path.
 
-Angular extends HTML with additional syntax that lets you insert dynamic values from your component.
+<!-- vale Angular.Google_WordListWarnings = YES -->
+
+Angular extends HTML with extra syntax that lets you insert dynamic values from your component.
```
Comment (bob-watson) [inline aio/content/guide/what-is-angular.md:71]: How about:
```suggestion
Angular adds syntax elements that extend HTML so you can insert dynamic values from your component .
```

File: aio/content/guide/what-is-angular.md [hunk 2]
```diff
@@ -106,7 +110,7 @@ The following is a combined example of Interpolation, Property Binding, and Even
     <code-pane header="hello-world-bindings.component.html" path="what-is-angular/src/app/hello-world-bindings/hello-world-bindings.component.html"></code-pane>
 </code-tabs>
 
-Add additional functionality to your templates through the use of [directives](guide/built-in-directives).
+Add extra features to your templates through the use of [directives](guide/built-in-directives).
```
Comment (bob-watson) [inline aio/content/guide/what-is-angular.md:113]: If you're adding them, they're additional, and extra. :smile:
```suggestion
Add features to your templates by using [directives](guide/built-in-directives).
```

File: aio/content/guide/what-is-angular.md [hunk 3]
```diff
@@ -151,53 +155,64 @@ For more information about dependency injection and Angular, see the [Dependency
...
+For more information about the Angular CLI, see the [Angular CLI Reference](cli) section.
 
 <a id="1p-libraries"></a>
 
 ## First-party libraries
 
-The section, [Angular applications: The essentials](#essentials), provides a brief overview of a couple of the key architectural elements you'll use when building Angular applications.
-But the many benefits of Angular really become apparent when your application grows and you want to add additional functions such as site navigation or user input.
+The section, [Angular applications: the essentials](#essentials), provides a brief overview of a couple of the key architectural elements that are used when building Angular applications.
+The many benefits of Angular really become clear when your application grows and you want to add more functions such as site navigation or user input.
```
Comment (bob-watson) [inline aio/content/guide/what-is-angular.md:182]: How about:
```suggestion
The many benefits of Angular really become clear when your application grows and you want to add functions such as site navigation or user input.
```
`more` (or `additional`) isn't necessary.

**Review by bob-watson — CHANGES_REQUESTED** [review]
Thanks for the edits!
They look great. I had just a couple of suggestions.

**Review by bob-watson — APPROVED** [review]
LGTM

reviewed-for: global-docs-approvers

@AndrewKushnir it's ready for your review and merge. Thanks!

**Review by jessicajaniuk — APPROVED** [review]
LGTM

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 547b45a9da631dcf764979cd1837bc1f63c39721.

**Pull request closed without merging**